UnityProject/Assets/Scripts/Player/Mind.cs

```diff
@@ -100,7 +110,8 @@ public IPlayerPossessable PlayerPossessable
 	/// </summary>
 	private Dictionary<string, object> properties = new Dictionary<string, object>();
 
-	public bool NonImportantMind = false;
+
+	[SyncVar] public bool NonImportantMind = false;
```

Comment:
wouldn't it be better to just have different types of minds so we can add filtering for them in the future? this looks like a quick hack to a single problem that we have elsewhere

Reply:
seems fine to bung all the data into one mind, since any generic behaviour should be defined by antagonist/objectives/body/spells

Reply:
you misuderstood what I said. I meant that it would be better to tag what kind of mind are we dealing with instead of just saying "is this mind important or not"

Reply:
The problem is what's the difference between a player monkey and a monkey cube monkey?